Classes/Views/PBFileChangesTableView.m

```diff
@@ -15,11 +15,11 @@ @implementation PBFileChangesTableView
 
 - (NSMenu *)menuForEvent:(NSEvent *)theEvent
 {
-	if ([self delegate]) {
+	if(self.indexController) {
```

Comment:
Maybe redeclare `-(id)delegate` as `-(PBGitIndexController *)delegate` in the header if that's what you're after ?